config/src/config/secure_backend_config.rs

```diff
@@ -195,6 +199,7 @@ vault:
     server: "127.0.0.1:8200"
     token:
         from_config: "test"
+    network_timeout_ms: 5000
```

Comment:
here

Reply:
Relevant if we're parsing the config from text;  either the field is present in vault_config in which case we should expose the value or not.